event_seat_reserve/models/event_registration.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+import logging
+
+from odoo import fields, models
+from odoo.tools import config
+
+_logger = logging.getLogger(__name__)
+
+
+class EventRegistration(models.Model):
+    _inherit = "event.registration"
+
+    state = fields.Selection(
+        selection_add=[("reserved", "Reserved"), ("open",)],
+        ondelete={"reserved": "set default"},
+    )
+
+    def action_set_reserved(self):
+        if config["test_enable"] and not self.env.context.get(
+            "test_event_seat_reserve"
+        ):
+            _logger.info("Test mode is enabled, you cannot reserve a registration.")
+            return True
```

Comment:
Why do you need this?

Reply:
Some tests are failing on the two others PR's because when create a sale, normally the `event. registration` still on `draft` state but it's no more the case with these modules. So to avoid failing...